File: pset6/mario.py
import cs50

print("Height: ", end="")
while True:
    n = cs50.get_int()
    if n == 0:
        exit(1)
    if n >= 0 and n <= 23:
        break

for i in range(n):
    print(" " * (n - i - 1), "#" * (i + 1), "  ", "#" * (i + 1),sep='')

File: pset1/mario.c
#include<stdio.h>
#include<cs50.h>
int main(void){
    int n;
    
    do{
        printf("Height: "); 
        n = get_int();
       if(n==0) return 0;
        //printf("\n");
    }while(n<0 || n>23);
    for(int i=0; i<n;i++){
        for(int j = 0; j<n-i-1; j++) printf(" "); for(int j=0; j<i+1; j++) printf("#");
        printf("  ");
        for(int j = 0; j<i+1; j++) printf("#");
        printf("\n");
    }
    
}